brushfire-core/src/main/scala/com/stripe/brushfire/TreeTraversal.scala

```diff
@@ -148,10 +148,10 @@ case class DepthFirstTreeTraversal[K, V, T, A](order: (Random, List[Node[K, V, T
           Stream.empty
         case (leaf @ LeafNode(_, _, _)) :: rest =>
           leaf #:: loop0(rest)
-        case (split @ SplitNode(_)) :: rest =>
-          val newStack = split.findChildren(row) match {
+        case (split @ SplitNode(_, _, _, _, _)) :: rest =>
+          val newStack = split.evaluate(row) match {
             case Nil => rest
-            case node :: Nil => node :: rest
+            //case node :: Nil => node :: rest
```

Comment:
This is here to avoid calling `order` in the common case. Since there is only 1 order for a singleton list, we know what the result would be for any `order` function (which is not allowed to change the size).  Mainly, our `order` functions are not always very, uh, speedy, so if we could avoid them with a simple `if`, that is desirable.